deploy/clowdapp.yaml

```diff
@@ -586,6 +586,116 @@ objects:
             cpu: ${{CPU_REQUEST_LISTENER}}
             memory: ${{MEMORY_REQUEST_LISTENER}}
 
+    - name: evaluator-recalc-new-service
+      minReplicas: ${{REPLICAS_EVALUATOR_RECALC_NEW}}
+      webServices:
+        public:
+          enabled: false
+        private:
+          enabled: false
+        metrics: {}
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        command:
+          - /engine/entrypoint.sh
+          - evaluator-new
+        env:
+        - name: POSTGRESQL_USER
+          value: ve_db_user_evaluator
+        - name: POSTGRESQL_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: evaluator-db-passwd
+              name: vulnerability-db-passwds
+        - name: EVALUATOR_TOPIC
+          value: vulnerability.evaluator.recalc
+        - name: VMAAS_VULNERABILITIES_API
+          value: /api/v3/vulnerabilities
+        - name: LOGGING_TYPE
+          value: OPENSHIFT
+        - name: LOGGING_LEVEL_APP
+          value: DEBUG
```

Comment:
This should not be hardcoded DEBUG, but I'm ok to keep it like this for some time.

Reply:
I want to keep it debug just for the sake of stage, when we deploy to prod I can return it to INFO back